src/gui/ui/gotoDlg.ui

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There is a lot of empty space in the widget, you might want to find a way to remove it